modules/network-observability-pktdrop-overview.adoc

```diff
@@ -21,5 +21,14 @@ When packet drop tracking is enabled, you can see the following metrics represen
 * Top X dropped cause
 * Top X flow dropped rates stacked with total
 
+Two kinds of packet drops are detected by Network Observability: host drops and OVS drops. If any flows are dropped, they are shown in the side panel of the Traffic flow table. Host drops are prefixed with `SKB_DROP` and OVS drops are prefixed with `OVS_DROP`, and a link to a description of the drop type is also available in the side panel. Examples of host drop reasons are as follows:
```

Comment:
The same comment as above, no need of the second line

Reply:
I can see how the previous one wasn't needed because I basically said right before that more info about packet drops is shown in the side panel. But here, that previous mention of the side panel isn't there. DO you see what I mean? Maybe its still an obvious thing that needs to be removed 🤷‍♀️